workflow/controller/controller.go

```diff
@@ -436,10 +436,9 @@ func (wfc *WorkflowController) initManagers(ctx context.Context) error {
 	return nil
 }
 
-func (wfc *WorkflowController) runConfigMapWatcher(stopCh <-chan struct{}) {
-	defer runtimeutil.HandleCrash(runtimeutil.PanicHandlers...)
+func (wfc *WorkflowController) runConfigMapWatcher(ctx context.Context, stopCh <-chan struct{}) {
```

Comment:
Minor: accepting both `ctx` and `stopCh` is a bit redundant, since you can derive the latter from `ctx.Done()`